pages/list/[listId].tsx

```diff
@@ -85,55 +99,84 @@ export const ListPage: NextPage = () => {
       setFilteredServices([...filteredServices, serviceToAdd])
       setSelectFilterValues([...selectFilterValues, serviceToAdd])
     }
-
   }
 
   const removeFilter = (serviceToRemove: Service) => {
-    setFilteredServices(filteredServices.filter((service) => service.id !== serviceToRemove.id))
-    setSelectFilterValues(selectFilterValues.filter((service) => service.id !== service.id))
+    setFilteredServices(
+      filteredServices.filter((service) => service.id !== serviceToRemove.id)
+    )
+    setSelectFilterValues(
+      selectFilterValues.filter((service) => service.id !== service.id)
+    )
   }
 
-
   return (
-
-    <VStack w='100%' minH='calc(100vh - 96px)' h='100%' spacing={0} px={24}  >
+    <VStack w="100%" minH="calc(100vh - 96px)" h="100%" spacing={0} px={24}>
       <ServiceListProvider value={serviceListHandler}>
-        <Stack w='100%' height='100%' display={{ base: 'none', md: 'inherit' }}>
-
-          <HStack pt={16} pb={8} justifyContent='space-between' w='100%' >
-            <Heading size='2xl' textAlign='center' >
+        <Stack w="100%" height="100%" display={{ base: 'none', md: 'inherit' }}>
+          <HStack pt={16} pb={8} justifyContent="space-between" w="100%">
+            <Heading size="2xl" textAlign="center">
               {listName}
             </Heading>
             <LinkBox>
-            <LinkOverlay _hover={{ textDecoration: 'underline' }}>
-            <Button colorScheme='teal'> Create a new List <PlusSquareIcon mx={1} /> </Button>
-            </LinkOverlay>
+              <LinkOverlay
+                href="/create-list"
+                _hover={{ textDecoration: 'underline' }}
+              >
+                <Button colorScheme="teal">
+                  {' '}
+                  Create a new List <PlusSquareIcon mx={1} />{' '}
+                </Button>
+              </LinkOverlay>
             </LinkBox>
           </HStack>
 
-          <HStack w='100%' justifyContent='space-between' >
-            <SearchBar baseData={services} setData={() => { }} searchFields={['name', 'description']} w={{ base: '100%', sm: '60%' }} mb='24px' />
+          <HStack w="100%" justifyContent="space-between">
+            <SearchBar
+              baseData={baseServices}
+              setData={setServices}
```

Comment:
Search didn't work on the list page before because `setData` was an empty function